api/messages.proto

```diff
@@ -103,6 +103,14 @@ message DoubleRangeFilter {
 
   // Minimum value.
   double min = 3;
+
+  enum Exclude {
+    NONE = 0;
+    MIN = 1;
+    MAX = 2;
+    BOTH = 3;
+  }
+  Exclude exclude = 4;
```

Comment:
Add a comment explaining what this means, with comments on each of the enum values.

Reply:
Added a comment + beta notice